plynx/web/user.py

```diff
@@ -23,6 +25,31 @@ def get_auth_token():
     })
 
 
+@app.route('/plynx/api/v0/register', methods=['POST'])
+@handle_errors
+def post_register():
+    email = request.headers.get('email').lower()
+    username = request.headers.get('username').lower() 
+    password = request.headers.get('password')
+
+    dic = validate_user(
+        email=email, 
+        username=username, 
+        password=password, 
+        confpassword=request.headers.get('confpassword')
+    )
+
+    if dic['success']:
+        user = run_create_user(email, username, password)
+        access_token = user.generate_access_token()
+        refresh_token = user.generate_refresh_token()
+        dic['access_token'] = access_token.decode('ascii')
+        dic['refresh_token'] = refresh_token.decode('ascii')
+        return dic
+
+    return dic
```

Comment:
please use `JSONEncoder().encode` in the end of every endpoint
https://github.com/plynx-team/plynx/blob/master/plynx/web/node.py#L175
And use field `status` as well. It will ensure that all of the endpoints are standardized